modern-fortran.sublime-syntax

```diff
@@ -548,6 +551,47 @@ contexts:
     - match: (?i)\b(end)\b\s+(?=interface)
       scope: keyword.declaration.interface.interface.fortran
 
+  acc:
+    - match: '{{firstOnLine}}(?i)(\!\$acc)'
+      scope: keyword.control.directive.fortran
+      push: acc-line
+
+  acc-line:
+    - include: acc-continuation
+    - include: acc-directives
+    - include: acc-clauses
+    - include: separators
+    - include: operators
+    - match: (?i)(default)
+      scope: keyword.control.directive.fortran
+      push: acc-default
+    - include: match-variable
+    - match: '\n'
+      pop: true
+
+  acc-directives:
+    - match: (?i)({{accDirectives}})
+      scope: keyword.control.directive.fortran
+
+  acc-clauses:
+    - include: acc-continuation
+    - match: (?i)({{accClauses}})
+      scope: keyword.control.directive.fortran
+
+  acc-default:
+    - include: acc-continuation
+    - match: (?i)(none|present)
+      scope: support.constant.acc
+    - match: '{{parenthesisEnd}}'
+      pop: true
```

Comment:
```suggestion
    - include: acc-continuation
    - include: eol-pop
    - match: (?i)(none|present)
      scope: support.constant.acc
    - match: '{{parenthesisEnd}}'
      pop: true
```
Include end-of-line pop. This will let us avoid incorrect highlighting on subsequent lines in the case of open incomplete parentheses: `!$acc ... default(`